packages/textfield/src/textfield.css

```diff
@@ -153,6 +153,17 @@ textarea {
     box-shadow: none;
 }
 
+:host([multiline][rows]) .input {
+    block-size: auto;
+    resize: none;
+}
+
+:host([multiline][rows="1"]) .input {
+    block-size: auto;
+    resize: none;
```

Comment:
These lines should already be covered by line 157/8 from above. Can we remove?

Reply:
Yes, I cleaned it up. Thanks for spotting it